spark/src/test/scala/org/apache/comet/exec/CometColumnarShuffleSuite.scala

```diff
@@ -55,7 +50,10 @@ abstract class CometShuffleSuiteBase extends CometTestBase with AdaptiveSparkPla
     super.test(testName, testTags: _*) {
       withSQLConf(
         CometConf.COMET_COLUMNAR_SHUFFLE_ASYNC_ENABLED.key -> asyncShuffleEnable.toString,
-        CometConf.COMET_EXEC_SHUFFLE_SPILL_THRESHOLD.key -> numElementsForceSpillThreshold.toString) {
+        CometConf.COMET_EXEC_SHUFFLE_SPILL_THRESHOLD.key -> numElementsForceSpillThreshold.toString,
+        CometConf.COMET_EXEC_ENABLED.key -> "false",
```

Comment:
I thought about this too but not exactly sure. I think it doesn't look necessary. @viirya do you remember why?

Reply:
Hmm? Did we have this before? Looks like it is added by this PR, no?

Reply:
Oh, there are many tests for columnar shuffle disabling it.

Reply:
I think some are intentional as some bugs only happen if comet exec is disabled. Some may not necessary, and possibly because it simplifies the cases (no native operators).

Reply:
Yea, I forgot why we did that.

Reply:
I see, I'll keep it then.